Sources/SwiftSyntax/RawSyntax.swift

```diff
@@ -251,6 +251,13 @@ extension RawSyntax {
     }
   }
 
+  func accumulateTrailingTrivia(_ pos: AbsolutePosition) {
```

Comment:
This has since been removed on `master` in favour of `trailingTriviaLength ` so I don't think we should add it back here. Should be part of https://github.com/apple/swift-syntax/commit/fc313fc29c33baf1d56fc0c965964c2e7c517e50

Reply:
Done.